### DISCUSSION TITLE
Eggshell black hole universe

### DISCUSSION THREAD

**Discussion by ExpEarth** [a-cosmology-group/acg#204] [Cosmology Models]
As I've mentioned a lot, I have a new cosmology model based on the idea of a black hole universe.  I have had difficulties getting my model accepted at journals and so I decided to post a preprint of it:

https://www.preprints.org/manuscript/202402.0712/v1

The PDF is here: 
[In an eggshell preprints202402.0712.v1.pdf](https://github.com/a-cosmology-group/acg/files/14241253/In.an.eggshell.preprints202402.0712.v1.pdf)

You can get the main idea of the model from Fig. 1.  In my model, 95% of the universe's matter is in a thin plasma shell at the Hubble radius, which is also the Schwarzschild radius.  The other 5% is the galaxies and gas/dust in the interior.  With this configuration I propose there is a CMB cycle which drives gravity and a reverse process linked to the cosmological constant.  The cycle depends on the CMB at high redshifts being at higher temperatures than the local CMB.  The remote CMB always has a higher temperature in my model and is about 29 K at the shell.   It also depends on a specific model of spacetime, where spacetime is photonic in nature.  This allows light energy to be absorbed into spacetime in the Hubble redshift and be returned to CMB photons in a Hubble blueshift, thus giving rise to gravity and lambda.

I'd appreciate it if you would have a look at it and let me know your reactions.  I would like feedback on any errors I've made, big or small, and also what points need further clarification or expansion.

Lastly, thanks to everyone in ACG for all the valuable discussions here, especially Louis.  I know many of you won't like this model, but thanks anyway!

**Comment by HanDeBruijn** [comment]
@ExpEarth .
Don't blame me for skipping all of your beautiful proze and jumping immediately to formulas, as I usually do.
Employed here and now are the following ones.

$$
u = \frac{4\sigma}{c}T^4 \qquad (2)
$$

$$
h = \frac{H_0}{\rho c} \qquad (15) \quad \quad \mbox{with} \quad \rho = \rho_c = 3H_0^2/8\pi G
$$

It follows that

$$
h = \frac{8\pi G}{3 H_0 c}
$$

The following claim is made (again) in your paper.

$$
G = \frac{u h^2}{6} \qquad (18)
$$

Serious, Matt: **nobody can calculate the gravitational constant**, other than by a vicious circle.
Therefore, let the IMHO proper sequence be presented below.

$$
u = \frac{6 G}{h^2} = \frac{6 G (3 H_0 c)^2}{(8 \pi G)^2} = \frac{27}{32 \pi^2}\frac{(H_0 c)^2}{G}
$$

In comparison, everybody has, and I have of course [my own theory](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm) too, with the following result.

$$
u = \frac{W^2.\mbox{VMT}.e^{-W}}{32\pi.wW}\frac{(H_0 c)^2}{G}
$$

Please note the **similarities** between the math in the two models: there is common physics in the constant $(H_0c)^2/G$.
Anyway, let's calculate and compare (with MAPLE).
```
Matt := evalf(27/(32*Pi^2));
W := 3.383634283;
VMT := int(exp(-x)/x^2,x=W..infinity);
wW := 2-exp(-W)*(W^2+2*W+2);
Han := evalf(W^2*VMT/(32*Pi*wW)*exp(-W));
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H_0 := 73.4; # Riess
H_0 := H_0*1000/Mpc;
c := 299792458;
u := Han*(H_0*c)^2/G;
sigma := 5.670374419*10^(-8);
Theta := evalf((c/(4*sigma)*u)^(1/4));
# Measured CMB (mean) temperature:

                         Theta := 2.764453562

u := Matt*(H*c)^2/G;
remote := evalf((c/(4*sigma)*u)^(1/4));

                        remote := 30.46148075
```
In concordance with, quoting Matt:
>The remote CMB always has a higher temperature in my model and is about 29 K at the shell.

**Note.** The following formula is also in the paper. I think it is at tension with the above one found for $u$.

$$
u_g = \frac{3c^2H_0^2}{16\pi G} \qquad (13) \qquad u_g \ne u = \frac{27}{32 \pi^2}\frac{(H_0 c)^2}{G}
$$

**Reply by ExpEarth** [reply]
Thanks for having a look at it, Han.  You certainly have a keen eye for equations!  But without the prose to go along with equations, no one can understand what the equations were meant to say.   That's one of the big problems with physics relative to other disciplines.  There are too may things one can hide in the equations.  I think Maxwell was the first one to cause problems, when he removed the aether structure he used to develop his 'Maxwell's equations' and said the equations alone were enough.  Einstein took the ball from there and the rest is sad history. 

On your specific point about the relationships between _G_ , _H_ and the mass absorption coefficient, _h_ , yes there is some circularity there.   But this can potentially be removed.  In (15) I say that _h_ = _H_ / (rho _c_ ).  Then I used the critical density to derive the value for _h_ .  As I show more generally for black holes, in (21), the cosmic critical density has exactly the same form as the generic black hole density.  From (20) we see that the black hole density is also related to _G_ , creating circularity again.  However, for black holes, as a class of objects, we can in most cases determine their masses directly through their gravitational effects on other masses (okay, we have _G_ in those interactions) and their radius can be directly determined in some cases too, as in supermassive black holes.  Thus we can have values for black hole density that do not actually depend on the value of _G_ and so the absorption coefficient _h_ can be uniquely specified.  That gives us a direct way of determining _G_ that does not depend on _G_ itself.  In Section (7) I argue that the value for _G_ is constant in all black holes in any case.  This is a relief, for otherwise one could not draw the connections between the universe and black holes that I have.

There are actually numerous issues that require greater coverage than I was able to do in the paper.  You may have noticed that the optical approach was not leaned on to such an extent as with my earlier papers.  It seems that gravity is a straight absorption process, governed by the temperature difference between the CMB and spacetime at various positions.  I also did not discuss the speed of light in black holes.  I have assumed in the paper that it is constant in black holes, yet that creates a gap with my earlier mechanism of light absorption (as with the Abraham-Minkowski dilemma, etc).  To me it seems that if _G_ is everywhere constant, most likely _c_ is too!

So this is not a done deal by any means.  It's a work in progress.  Yet I am 100% sure that the basic model I have here for the visible universe and black holes is fundamentally correct.

P.S.  On your last point comparing our CMB temperatures, I will reply to you later.  You raised this in another thread and I have to go back to it.

**Reply by HanDeBruijn** [reply]
My own theory in a nutshell. It is IMHO simpler than yours, conceptually speaking.
As I've repeatedly said, there is much **formal** resemblance between the two models.
- The universe is infinite and eternal. Better: there is NO universe as a whole.
- The Variable Mass Theory (VMT) is valid all over the place; no question about it.
- Geometry is Euclidean. There is no space-time curvature and no General Relativity.
- A finite **observable** black hole universe is present at any space-time "center".
- Black hole universes are equipped with a negative gravitational self energy field.
- The negative mass associated with that self evergy would annihilate all other mass
- if it were not that it is compensated with a positive mass/energy, which is the CMB.
- The CMB radiates inward at the rim of the black hole universe with intrinsic redshift.
- Temperatures are not redshifted; the associated energy/mass densities **_are_** redshifted.
- The CMB temperature is uniform &amp; the same inside a black hole / observable cosmos.

**Reply by ExpEarth** [reply]
There are indeed quite a few similarities in our two models, but some notable differences also!  It would take some time to go over all the points where we disagree, so let's just celebrate that we have some common points of agreement for now.

**Comment by ExpEarth** [comment]
I just heard back from another journal and they asked me to improve the article before they would send it for peer review.  Here is part of what they said:

"The paper explores some thermodynamic properties of the model, but completely ignores the dynamics of the shell.  In the standard cosmology, dynamics is encoded in a coupled system of equations for the gravitational field and a homogeneous and isotropic fluid. Since the author removes the homogeneity condition, one must solve Einstein's equations in the new setting to demonstrate viability of the new proposal. In particular, the author claims that the Newtonian analysis might be sufficient (without proving this statement), but even such an analysis is not performed apart from some simplistic discussion on page 9. The idea of this article is very interesting, but the claims about dynamics are not well justified. I recommend revising the manuscript by adding the counterparts of the standard cosmological equations (either in general relativity or in the Newtonian approximation) for the new setting before asking for a detailed peer review."

The problem is, I'm not the right one to do those things!  Other journals made the same general comments.

**Reply by HanDeBruijn** [reply]
>In the standard cosmology [ .. ] one **must** solve Einstein's equations [ .. ] to demonstrate viability

See? It is not possible with "peer reviewed" journals to deviate from _the standard cosmology_ or **not** to _solve Einstein's equations_. Surprised? I'm not. Science has developed into a dictatorship. Independent thinking is forbidden by the state :-(of affairs).
>The problem is, I'm not the right one to do those things!

No, the problem that you are still seeking sort of companionship with established cosmology, which maybe is not even desirable. Why not simply give up? And formulate your theory without compromising. It's better to have no friends than bad friends.

**Reply by ExpEarth** [reply]
I don't think it's unfair of them to ask me to describe the dynamics of the shell more.  At this point they probably don't realize the full implications of my model.

I would say it is very unwise to give up on publishing in mainstream journals.  That is where the theories need to be seen, in order to eventually get accepted.  The non-mainstream journals and outlets have a role, but it's mainly in getting us up to strength to publish in the bigger journals.

**Comment by HanDeBruijn** [comment]
## Gravitons

Source is: [Origin of Inertia](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/96836893X-rev.pdf) by Amitabha Ghosh. By the way, because of GR lip service being demanded, Amitabha is has been suffering from the same sort of criticism as you do.
Section 5.3 in his book is about gravitons. There we read that the energy of the graviton is given by

$$
E = E_0 \exp\left(-\frac{\kappa}{c}r\right) \qquad (5.5)
$$

Section 6.2 in the book is about photons. There we learn that the energy of the photon must be given by

$$
\frac{E}{E_0} = \exp\left(-\frac{\kappa}{c}x\right) \qquad (6.10)
$$

Here $x=r$ and for the quantity $\kappa$ it turns out that $\kappa = H_0 =$ the Hubble constant.
Anyway, we have the same distance dependent formula **for both** the energy of the graviton and the energy of the photon.
Is that a coincidence? Nope. According to the Variable Mass Theory it is not.
Variable mass explains both the the energy of the photon (by intrinsic redshift) as well as the apparent energy of the graviton. The latter is explained by the variable mass, and therefore a variable mass density, by itself. The reason is that the gravitational constant and mass are often in combination, called [gravitational parameter](https://en.wikipedia.org/wiki/Standard_gravitational_parameter), resulting in a $G$ that seems to vary.

Okay, now about your paper. You are repeatedly talking about
>graviton filaments of spacetime

Empirically, nothing can be shielded from gravitation. Such in contrast with photons: they can be shielded very easily.
There is not a single reason why gravitation should be propagated by particles called gravitons eventually. [Hypotheses non fingo](https://en.wikipedia.org/wiki/Hypotheses_non_fingo), [Occam's razor](https://en.wikipedia.org/wiki/Occam%27s_razor). [Particle Fever](https://en.wikipedia.org/wiki/Particle_Fever) should stop here!

**Reply by ExpEarth** [reply]
I use the term gravitons in a specific sense that is different from what you would normally hear.  Gravitons are not the transmitters of the gravitational force.  They are the fundamental units of spacetime which can absorb photonic energy (increasing graviton content and causing gravitational attraction) and then release that same energy back to photons (decreasing the filaments' graviton content and thereby causing masses to separate).  The filaments are attached to masses and so you could also say that masses in a sense absorb photonic energy, as in some Le Sage models.  This could be confusing, but I have been consistent in my usage for over a decade now.  Annila et al have a similar description of gravitons, but their gravity mechanism is different from mine.

**Reply by HanDeBruijn** [reply]
Apologies for twisting your prose in my favor:
>Gravitons are [ .. ] causing gravitational attraction) and then release that same energy back to photons

Apart from any detailed mechanism that is: conversion of gravitational energy into electromagnetic energy.
Isn't that (approximately) the same as my saying: _Gravitational (self) energy + (Uniform) CMB energy = 0_ ?
Let's consider **your** model again, as if it were **my** model with the Variable Mass Theory as the **only** exception.
One of your assumptions is that radius and volume of the black hole universe are given by

$$
R_U = \frac{c}{H_0} = \frac{2GM}{c^2} \quad ; \quad V_U = \frac{4}{3}\pi R_U^3
$$

From the first the mass of the observable (black hole) universe can be calculated.

$$
M = \frac{c^3}{2G H_0}
$$

According to the paper, _the internal gravitational potential energy of the EU is_

$$
U_{EU} = -\frac{GM^2}{R_U} \qquad (7)
$$

Which is a factor of two different from the standard expression for the gravitational [**self** energy](http://www.alternatievewiskunde.nl/cosmology/nasa.htm) $\ne$ potential energy (?)

$$
U'_{EU} = -\frac{GM^2}{2R_U} \qquad (7.13)
$$

But okay, then you arrive correctly at an equation we have encountered [before](https://github.com/orgs/a-cosmology-group/discussions/204#discussioncomment-8447430).

$$
u_g = -\frac{U_{EU}}{V_U} = \frac{3}{16\pi}\frac{(cH_0)^2}{G} \qquad (13)
$$

While I would have preferred, still **without** taking into account the Variable Mass Theory:

$$
u'_g = \frac{3}{32\pi}\frac{(cH_0)^2}{G}
$$

Now equate CMB energy density to gravitational energy density and calculate.
```
Matt := evalf(3/(16*Pi));
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H_0 := 73.4; # Riess
H_0 := H_0*1000/Mpc;
c := 299792458;
u := Matt*(H_0*c)^2/G;
sigma := 5.670374419*10^(-8);
CMB := evalf((c/(4*sigma)*u)^(1/4));

                          CMB := 27.84423885

Han := evalf(3/(32*Pi));
u := Han*(H_0*c)^2/G;
CMB := evalf((c/(4*sigma)*u)^(1/4));

                          CMB := 23.41412064
```
The factor two shows some minor improvement when compared with the original of yours.
(Guess where a major improvement might come from)

**Reply by ExpEarth** [reply]
Your factor of 2 could indeed help matters.  For the Earth, the internal gravitational potential energy is _U_ = .6 _GM_^2/_R_^2.  For an object like a star the numerical term is sometimes just approximated as 1 and that's what I did.  This could be an area where I could improve the model.  It's interesting the numerical correspondence between our two models.  Thanks for your continuing interest, Han.
